### PR TITLE
Testing characters skills

### LINKED ISSUE #56: Test-case 20 Lose scenario of the game because of Blizzard

### LINKED ISSUE #57: Test-case 19 Win scenario of the game

### LINKED ISSUE #60: Test-case 16 Eskimo skill use

### LINKED ISSUE #61: Test-case 15 Polar explorer skill us

### LINKED ISSUE #76: Test-case 4 Edge of the map

### PR DESCRIPTION
**Pull request by annahramova97**
For Eskimo no new test method added because it's skill is properly tested in the testing of blow method. You can navigate in this tests using the comments.

### CODE REVIEW

**Comment by annahramova97** [comment]
For lose scenario I don't specify a special case because I tested it in other tests(blizzard,in water).
Other tests can't be done at this point.

File: core/src/main/java/com/rim/IceField/BlizzardTest.java [hunk 1]
```diff
@@ -22,14 +22,17 @@ void blow() {
...
         Blizzard.blow(playersList, game.getMap());
         assertEquals(4,e1.heatLevel);
         assertEquals(2,game.getMap().Icebergs[0][0].getAmountOfSnow());
-        assertEquals(2,game.getMap().Icebergs[0][1].getAmountOfSnow());
 
         assertEquals(4,e2.heatLevel);
         assertEquals(2,game.getMap().Icebergs[1][0].getAmountOfSnow());
 
+        //Test-case 16: Polar explorer skill
         e1.useSkill(game.getMap(),"east");
```
Comment (AirinB) [inline core/src/main/java/com/rim/IceField/BlizzardTest.java:37]: I guess Dinu here checks for the upper case letters

File: core/src/main/java/com/rim/IceField/PlayerBaseTest.java [hunk 2]
```diff
@@ -216,5 +217,105 @@ void eatFood() {
         assertEquals(5,p1.heatLevel);
      }
 
+     @Test
+     void polarSkill(){
+        //Test-case 15: Polar explorer skill
+         PlayerBase  p1 = new PolarExplorer();
```
Comment (AirinB) [inline core/src/main/java/com/rim/IceField/PlayerBaseTest.java:224]: Do you have the same setup in all of the tests? maybe you could use something like before each or before all, to create one setup.

**Review by AirinB — APPROVED** [review]

File: core/src/main/java/com/rim/IceField/BlizzardTest.java (hunk 1 above)
Comment (annahramova97) [reply core/src/main/java/com/rim/IceField/BlizzardTest.java:37]: Anyway with new changes this tests will be modified also. Will modify the staff tomorrow with all new changes.

File: core/src/main/java/com/rim/IceField/PlayerBaseTest.java (hunk 2 above)
Comment (annahramova97) [reply core/src/main/java/com/rim/IceField/PlayerBaseTest.java:224]: It really depends on the test case: number of chatacters, type of chaarcters, if we use game as an instances, but there are some common things of course, so before is a good suggestion. Will add that.

**Pull request merged**